### ISSUE TITLE
other ECUs ?

### ISSUE THREAD

**Issue by freeeflyer** [fr3ts0n/AndrOBD#89]
Hello,
Is there a way to access other ECU like mrs/srs or others ?
thanks

**Comment by fr3ts0n** [comment]
No, usually only emissions relevant ECUs of the drive train support the OBD protocol.

**Comment by freeeflyer** [comment]
Are you sure ? I accessed airbag and all other ECU with an android commercial closed source solution and my ELM327..

**Comment by ChrisS81** [comment]
My Car has 3 ECUs on a CAN Bus and i can connect to any of them using AndrOBD without issues. 
Sure, some if them are sending Nonsens data for some Codepoints but nonetheless i can connect to them.
The first one AndrOBD shows me is the Engine ECU. The second one is Transmission Control and the third is something like exhaust Processing IIRC (AdBlue system i guess)

Am 12. November 2019 07:55:38 MEZ schrieb Nicolas Zagulajew <notifications@github.com>:
>Are you sure ? I accessed airbag and all other ECU with an android
>commercial closed source solution and my ELM327..
>
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/fr3ts0n/AndrOBD/issues/89#issuecomment-552761023

-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by fr3ts0n** [comment]
> Are you sure ? I accessed airbag and all other ECU with an android commercial closed source solution and my ELM327..

Other ECUs may be accessed using different communication protocol(s) than OBD. 
These protocols  are only partially standardized and their data content is very specific to the vehicle manufacturer- and even vehicle type.

That's why these apps usually are closed source, specific to certain manufacturers and commercial.
Some of these even can operate on ELM327 adapters.

**Comment by fr3ts0n** [comment]
@ChrisS81 I agree, all three ECU's you can connect with AndrOBD are emissions relevant and obviously service OBD protocol. 
However, chances are poor to find an Airbag- or SRS-ECU which serves OBD protocol.

**Comment by enfranc1991** [comment]
555
